ethereum/spec/src/test/java/tech/pegasys/teku/spec/logic/common/helpers/MiscHelpersTest.java

```diff
@@ -231,6 +231,12 @@ public void committeeComputationShouldNotOverflow(int activeValidatorsCount, int
         });
   }
 
+  @Test
+  public void isFormerDepositReceiptMechanismDisabled_returnsFalse() {
```

Comment:
Is it possible to run it parameterized for all forks which are less than electra?

Reply:
I like it. It was pretty easy. Done.